aldryn_events/tests/frontend/integration/pages/alteli.js

```diff
@@ -0,0 +1,166 @@
+// ALDRYN TESTS LIBRARY
```

Comment:
@Marketionist no alteli.js here for now, we need to write vanilla tests using only protractor